src/brevitas/export/manager.py

```diff
@@ -269,7 +269,10 @@ def jit_inference_trace(cls, module: Module, args: Union[Tensor, QuantTensor, Tu
             module = module.eval()
             module.apply(cls.set_export_handler)
             # do a forward pass with the input to e.g. store input/output shapes
-            cls._cache_inp_out(module, *args)
+            if isinstance(args, Tuple):
```

Comment:
This doesn't look right, args could be a QuantTensor, which is a NamedTuple, and you don't wanna unpack that